GitCommands/Settings/AppSettings.cs

```diff
@@ -1914,6 +1920,12 @@ public static int RepoObjectsTreeSubmodulesIndex
             set => SetInt("RepoObjectsTree.SubmodulesIndex", value);
         }
 
+        public static int RepoObjectsTreeStashesIndex
+        {
+            get => GetInt("RepoObjectsTree.StashesIndex", 4);
```

Comment:
I have moved it to the top.
(The weird auto-reload of settings without UI update confused me once again. If another GE instance denies to work like for `Move up`, hit `F5`.) 
The `Stashes` tree is collapsed by default. So it does not consume much space but is at hand quickly.
(not requesting a change)

Reply:
The index is stored in settings, so it is safer to store new occurrences last.
Do you request a change?

Reply:
>> (not requesting a change)
>
> Do you request a change?

I just wanted to raise this point. Perhaps others vote for a change.

> The index is stored in settings, so it is safer to store new occurrences last.

Yes, else it might become difficult to handle.